src/modules/beamos/filesystem/mrb_check/root/usr/share/mrb_check.yaml

```diff
@@ -5,30 +5,24 @@ device_info:
   series: 2S
   model: MRBEAM2_DC
   model_force: MRBEAM2_DC
-  accepted_series:
```

Comment:
no need for these anymore? why?

Reply:
No I checked and this is not used, thats why I removed it.
If it would be used it should have been failed already as the last series was 2R